content/en/docs/concepts/data-collection.md

```diff
@@ -41,11 +45,180 @@ The Collector is made up of the following components:
 - <img width="32" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Exporters.svg"></img>
   `exporters`: Where to send received data; these can be push or pull based
 
-These components are enabled through `pipelines`. Multiple instances of
-components as well as pipelines can be defined via YAML configuration.
+These components are enabled through `pipelines` as mentioned earlier. Read further to learn
+more about the components.
+
+### Receivers
+The receiver is the first component in a pipeline. It receives data in several supported
+[data formats](https://opentelemetry.io/docs/concepts/signals), and converts this data into 
+an internal data format recognized by the collector. Receivers can be push or pull based.
+
+You can set multiple protocols for a single receiver and make them listen to different ports by default. 
+The table below shows supported receiver formats for each signal type:
+
+| Signal Source        | Traces             | Metrics            | Logs               |
+| :---                 |    :----:          |  :---:             |               ---: |
+| Host Metrics         |                    | :heavy_check_mark: |                    |
+| Jaeger               | :heavy_check_mark: |                    |                    |
+| Kafka                | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| OpenCensus           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| OpenTelemetry (OTLP) | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Prometheus           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| Zipkin               | :heavy_check_mark: |                    |                    |
+
+Learn how to configure receivers in the [configuration documentation](/docs/collector/configuration/#receivers).
+
+### Processors
+The job of the processor is to filter unwanted telemetry data and inject additional attributes to the data 
+before it is sent to the exporter. While receivers and exporters, the capabilities of processors differ immensely
+from one processor to the other. Processors are run on data between being received and being exported.
+While processors are optional, [some are
+recommended](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor#recommended-processors).
+
+The table below shows some currently supported processors, and the signals they
```

Comment:
I think it would be better to not include this paragraph + table. There should arguably be an authoritative table of processors, either in the docs or in the collector repository. But I think we can tackle that orthogonally to this PR because it would involve ongoing contributions by collector contributors/maintainers to keep it up to date.